catalog/app/index.html

```diff
@@ -26,6 +26,7 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <!-- styles for Jupyter Notebooks -->
     <link href="/ipynb.css" rel="stylesheet">
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=Intl%2Cdefault%2CResizeObserver%2CBlob%2Cfetch%2Ces2016%2Ces2017%2Ces2018%2Ces2019%2Ces2015"></script>
```

Comment:
The slow download will slow all stuff below from downloading.
I suppose that the application's javascript will be inserted below and will be downloaded after plolyfill.io.

Maybe, `npm` package is safer?

Reply:
> The slow download will slow all stuff below from downloading.

brief testing shows that polyfill.io works really fast (+ caching), and it's probably more reliable than any of our homegrown solutions would be

> I suppose that the application's javascript will be inserted below and will be downloaded after plolyfill.io.

that's the point, yes, to load all the polyfills before executing the app code.

> Maybe, npm package is safer?

safer how?

the main point of using polyfill.io vs polyfills included in the bundle is smaller bundle size and automatic polifylling based on user-agent.
we could run the polyfill service ourselves if we dont trust polyfill.io enough, but this would require running an extra container and i'm not sure we need that

Reply:
Polyfill.io will be down one day, and will block our application. If you think, that effort to avoid this is much bigger than that risk, I'm totally fine with that.

Reply:
> If you think, that effort to avoid this is much bigger than that risk

yep, i think so, tho we can backlog revisiting the polyfilling setup later